addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -132,6 +150,24 @@ QUnit.module('web_editor', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('notebooks defined inside HTML field widgets are ignored when calling setLocalState', async function (assert) {
+            assert.expect(0);
+
+            var form = await testUtils.createView({
+                View: FormView,
+                model: 'note.note',
+                data: this.data,
+                arch: '<form>' +
```

Comment:
also, here, you can use template strings

Reply:
it's copy-pasted too